src/main/java/org/espeakng/jeditor/gui/Splashscr.java

```diff
@@ -0,0 +1,100 @@
+package org.espeakng.jeditor.gui;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Font;
+import java.awt.Graphics2D;
+import java.awt.SplashScreen;
+import java.awt.geom.Rectangle2D;
+import java.awt.geom.Rectangle2D.Double;
+
+public class Splashscr {
+	static SplashScreen mySplash = SplashScreen.getSplashScreen();
+	static Graphics2D splashGraphics = mySplash.createGraphics();
+	static Dimension ssDim = mySplash.getSize();
+	static int height = ssDim.height;
+	static int width = ssDim.width;
+    static Double splashTextArea = new Rectangle2D.Double(15., height*0.88, width * .45, 32.);
+    static Double  splashProgressArea = new Rectangle2D.Double(width * .55, height*.92, width*.4, 12 );
+	  static void splashInit() {
+		    if (mySplash != null)
+		    { 
+		        Font font = new Font("Dialog", Font.PLAIN, 14);
+		        splashGraphics.setFont(font);
+		        // initialize the status info
+		        splashText("Starting");
+		        splashProgress(0);
+		    }
+		} // initialize splash overlay drawing parameters
+      static void appInit() {
+    	    for(int i=1;i<=10;i++)
+    	    {
+    	        int pctDone = i * 10;
+    	        splashText("Doing task #" + i);
+    	        splashProgress(pctDone);
+    	        try
+    	        {
+    	            Thread.sleep(1000);
+    	        }
+    	        catch (InterruptedException ex)
+    	        {
+    	            // ignore it
```

Comment:
1. Initialize logger for this class
2. `logger.warn(ex);`
3. `Thread.currentThread().interrupt();`